code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -7,6 +7,7 @@
 	default_language = "Galactic Common"
 	flags = NO_BREATHE | NO_PAIN | NO_BLOOD | NO_SCAN
 	dietflags = DIET_OMNI		//golems can eat anything because they are magic or something
+	reagent_tag = PROCESS_DUO	//golems only process a select few chems like acid and hellwater because magic?
```

Comment:
Trauma and Burn packs should still work on them, but I could have sworn they had a natural regeneration. I'll look into this more, and can give them PROCESS_ORG, PROCESS_SYN, or even both since they are magic.